src/components/custom-aggrid/custom-aggrid-header.jsx

```diff
@@ -322,7 +326,7 @@ const CustomHeaderComponent = ({
                 >
                     {isAutoCompleteFilter ? (
                         <Autocomplete
```

Comment:
Using autocomplete for a boolean filter is not suitable. It would be better to create a new component, such as a Select dropdown, to handle the boolean filter type.